docs/development/release.md

```diff
@@ -63,7 +66,9 @@ Note: This is for 1.10.3 or later.
 
 ### 6. Create a pull request against the minor version branch
 
-Create a PR against the minor version branch. Verify that the Bamboo build for the PR succeeds and then merge to the minor version branch.
+Push the release branch to GitHub.
+Create a PR against the minor version base branch. Verify that the Bamboo build for the PR succeeds and then merge to the minor version base branch.
+Do not delete your release branch after merging.
```

Comment:
Doesn't look like we have deleted any of them, although I'm not sure why as they have been merged to their base branches. I put this in because there was no clarity, but do we have a policy?

Reply:
You could delete the minor branch I think since it's tagged anyway.